delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/ResourcesRoutes.scala

```diff
@@ -166,6 +166,20 @@ final class ResourcesRoutes(
                           )
                         }
                       },
+                      (pathPrefix("refresh") & put & pathEndOrSingleSlash) {
+                        operationName(s"$prefixSegment/resources/{org}/{project}/{schema}/{id}/source") {
```

Comment:
We aren't taking revision as an argument. We will see later that revision is needed, so we take it from the record we load

Reply:
replace `/source` with `/refresh` in operation name

Reply:
Not sure how this happened :)